Library/Formula/websocketd.rb

```diff
@@ -0,0 +1,24 @@
+class Websocketd < Formula
+  homepage "http://websocketd.com"
+  url "https://github.com/joewalnes/websocketd/archive/v0.2.10.tar.gz"
+  sha1 "4387c39aaf77e1bb5b6991358f02611a7b5dac1e"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    mkdir_p "src/github.com/joewalnes"
+    ln_s "../../../", "src/github.com/joewalnes/websocketd"
+
+    system "go", "get", "./src/github.com/joewalnes/websocketd"
```

Comment:
Nope, it fetches itself from a link back to the current directory. I don't really know enough about the go build system to be 100% sure, but I think it's because `websocketd` is split up into `websocketd` and `libwebsocketd`. For some reason the `libwebsocketd` is linked as `"github.com/joewalnes/websocketd/libwebsocketd"` in `main.go`. Since this is already in available locally we want to use it straight up, also to ensure that we are using the same version of both parts.

I'm guessing that it's done this way so that other programs can use just `libwebsocketd`.

I got the code from the `Makefile` in `websocketd`, but I didn't use it straight up because it fetches it's own version of `go` and installs it locally. I think this is done because other package managers dosen't have the same nice policies as brew does and ships really old versions of `go`.

Perhaps @asergeyev or @joewalnes could explain this better.